src/main/java/cms/gov/madie/measure/services/ActionLogService.java

```diff
@@ -18,7 +19,11 @@ public class ActionLogService {
   private final MeasureActionLogRepository actionLogRepository;
 
   public boolean logAction(
-      final String targetId, Class targetClass, final ActionType actionType, final String userId) {
+      final String targetId,
+      Class targetClass,
+      final ActionType actionType,
+      final String userId,
+      final String... additionalActionMessage) {
```

Comment:
not sure these 3 dots

Reply:
I was using these dots to make it optional

Reply:
ah..okay, thanks!